frontends/web/src/routes/account/send/send.tsx

```diff
@@ -279,6 +280,7 @@ class Send extends Component<Props, State> {
                     proposedFee: result.fee,
                     proposedAmount: result.amount,
                     proposedTotal: result.total,
+                    note: '',
```

Comment:
what's this for? this should not reset the state.

with this, the note is deleted if you modify the amount for example.

Reply:
fixed